turbinia/turbiniactl.py

```diff
@@ -957,10 +957,22 @@ def process_evidence(
             'jobs allow/deny lists, yara rules or a patterns file separately.')
         raise TurbiniaException(msg)
 
+      # Find the recipe file with the following precedence:
+      #   1. An absolute path to a recipe file provided with --recipe_path.
+      #   2. A combination of the RECIPE_FILE_DIR config option and the
```

Comment:
We should probably change the default `RECIPE_FILE_DIR = None` in the default/template config.

Reply:
Done